### ISSUE TITLE
Add hillside

### ISSUE THREAD

**Issue by azzamsa** [jhelvy/splitKbCompare#85]
Please complete this checklist and submit your issue - I will do my best to add your keyboard as soon as I can:

- [X] Keyboard name: Hillside
- [X] Url to vector graphic (e.g. svg, pdf, etc.) of switch plate:https://github.com/mmccoyd/hillside/blob/main/doc/image/hillside-switches_left.svg
- [X] Min and max number of keys: 42 to 48 keys 
- [X] Url to keyboard project source (if available): https://github.com/mmccoyd/hillside
- [ ] Url to store to buy keyboard (if available):

**Comment by azzamsa** [comment]
It is already added here: https://github.com/jhelvy/splitKbCompare/pull/71

But doesn't shows up.

![image](https://user-images.githubusercontent.com/17734314/170616751-99481f9e-2a83-4ce4-ae11-864d28c5e2e2.png)

**Comment by jhelvy** [comment]
Yes someone added the basic information for the hillside, but they didn't include the svg file in the overlay.ai file, so I still will need to add it before it will show up. As you can see from the backlog of issues, there are quite a few more keyboards that need to be added, but times are very much anything but normal right now, and this project is unfortunately quite far back on my todo list. So it'll have to wait for now. I never could figure out a better way to make it easier to add new keyboards. It's just a rather time-consuming and manual process for now, making me the chokepoint. In the future, I may write a more detailed set of instructions so that someone with Illustrator can edit the file themselves and create a PR.

**Comment by jhelvy** [comment]
There are certainly much better implementations than the current version. What I'd much rather have is a way to have an app where the user can upload a svg of the plate and manually align it themselves (drag & scale interface, aligning to some standard size box). This could probably be done with just javascript, but I'm not a developer and don't know where to begin. And I think most people in this community who might know how to do something like this probably have a lot of other things on their plates too.
